presto-orc/pom.xml

```diff
@@ -61,6 +61,11 @@
             <artifactId>stats</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
The pros to keep jsonCodec in TracerFactory is we don’t need to create a jsonCodec object whenever an event is emitted. If we use jsonCodec for operator level events it may take lots of resources for large queries. Also we will not introduce the dependencies to connector packages which don’t use jsonCodec currently.
The cons would be it makes it a little bit hard to read. 
I've updated this by creating jsonCodec object in TracerFactory for now.